activerecord/lib/active_record/transaction.rb

```diff
@@ -5,16 +5,29 @@
 module ActiveRecord
   # Class specifies the interface to interact with the current transaction state.
   #
-  # It can either map to an actual transaction or represent the abscence of a transaction.
+  # It can either map to an actual transaction/savepoint, or represent the
+  # abscence of a transaction.
```

Comment:
```suggestion
  # absence of a transaction.
```
That one is on me...